inst/rmd/h/navigation-1.1/tabsets.js

```diff
@@ -121,15 +128,29 @@ window.buildTabsets = function(tocID) {
       tab.detach().appendTo(tabContent);
     });
 
-    // set active tab
-    $(tabList.children('li')[activeTab]).addClass('active');
+    // activate tab (note users can add an .active class)
+    var navLinks = window.BS3_NAV ? tabList.children('li') : tabList.find(".nav-link");
+    $(navLinks[activeTab]).tab("show");
+
+    // make sure active content is shown on load
     var active = $(tabContent.children('div.section')[activeTab]);
     active.addClass('active');
     if (fade)
-      active.addClass('in');
+      active.addClass(window.BS3_NAV ? 'in' : 'show');
```

Comment:
Verify that this works with 4-3. And try `.in { @extend .show; }`

Reply:
Oh, whoops, this isn't currently working with `4-3`, but `.in { @extend .show; }` does appear to fix it

Reply:
The actual fading doesn't work for some reason though

Reply:
PR started here https://github.com/rstudio/bootsass/pull/9